### PR TITLE
Add JWT encoding & decoding for python API

### PR DESCRIPTION
**Pull request by Primdahl26**

### CODE REVIEW

File: api/app/main.py [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
-from .document import client, User, UserCreate
+from .document import client, User, UserRegister, UserBase
+from pymongo.errors import DuplicateKeyError
+from .utils import fatsms_send_sms, generate_token
+import jwt
+
 
 app = FastAPI()
 
+secret = "6horse9"
```
Comment (AndreasPB) [inline api/app/main.py]: Put in .env pls 🗡️

File: api/app/utils.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+def generate_token():
+    return "".join(
+        random.choice(string.ascii_uppercase + string.ascii_lowercase + string.digits)
+        for _ in range(4)
+    )
+
+
+async def fatsms_send_sms(message: str, to_phone: str):
+    url = "https://fatsms.com/send-sms"
```
Comment (AndreasPB) [inline api/app/utils.py]: Would prob make sense to also put in config.py, but not .env

**Review by AndreasPB — APPROVED** [review]
🐍

File: api/app/main.py (hunk 1 above)
Comment (Primdahl26) [reply api/app/main.py]: Done

File: api/app/utils.py (hunk 2 above)
Comment (Primdahl26) [reply api/app/utils.py]: Done :)

**Pull request merged**